### ISSUE TITLE
When newly promoted primary node also fails, third node does not work.

### ISSUE THREAD

**Issue by salmanahmed-postgresql** [stormatics/pg_cirrus#65]
In the event of a subsequent failure of the newly promoted primary, the third node fails to function.

**Comment by salmanahmed-postgresql** [comment]
When pgpool is started the status of all nodes is as following

```
2023-07-20 09:30:08.961: main pid 2582: LOG:  node status[0]: 1
2023-07-20 09:30:08.962: main pid 2582: LOG:  node status[1]: 2
2023-07-20 09:30:08.962: main pid 2582: LOG:  node status[2]: 2
```

It is observed that after failover the status of nodes changes to following
```
2023-07-20 09:37:48.587: sr_check_worker pid 2673: DEBUG:  node status[0]: 0
2023-07-20 09:37:48.587: sr_check_worker pid 2673: DEBUG:  node status[1]: 1
2023-07-20 09:37:48.588: sr_check_worker pid 2673: DEBUG:  node status[2]: 0
```

**Comment by salmanahmed-postgresql** [comment]
As per the suggestion from pgpool-hackers, the following property was added to pgpool.conf file for issue #4 
```
database_redirect_preference_list = '.*:standby'
```

When pgpool was started using ```backend_weight0=0```, ```backend_weight1=0.5``` and  ```backend_weight2=0.5```, after failover pgpool was stopped and the value of backend_host1 was set to 0 and value of backend_host2 was set to 1 and pgpool was restarted. In this setting, after the second failover the third node keeps working.


When pgpool was started using  ```backend_weight0=1```, ```backend_weight1=0.5``` and  ```backend_weight2=0.5```, after failover pgpool was stopped and started again. In this setting the third node kept on working

**Comment by salmanahmed-postgresql** [comment]
This issue has been fixed in PR #114  along with Issue #4.
